### ISSUE TITLE
0781 VBA consent page

### ISSUE THREAD

**Issue by jinaryu** [department-of-veterans-affairs/va.gov-team#89783]
Collaborating with Eileen Chi from Benefits Decision Reviews team on [VBA consent page](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=4468-78852&t=PDcH4fv1p5g5YouU-1), as they are also working on the same page in the supplemental claims form

- [x] Review notes from meeting with VHA and VBE SMEs on 6/28
- [x] Respond to comments
- [ ] Finalize page content with CAIA

**Comment by jinaryu** [comment]
Notes from 6/28 meeting

How does this benefit the veteran?
- unclear because it varies from veteran to veteran
- claim process is stressful esp for those with MST, so there may be benefit for a healthcare provider to be aware of this
- ex. provider's awareness, connect to offer additional support
- but this doesn't promise anything from the provider's end. some may act, some may not

What would the veteran be considering?
- if it'd be valuable for their VHA provider to be aware of these upcoming events
- what exactly happens when a notif is sent to VHA?
- put indicator in medical record to let provider know one of those events are scheduled to occur
- currently, it'll be a chart note but the permanent solution is different indicator. (don't say it's a chart note, this is temporary)
- indicator won't say MST, but currently indicator is only MST
- visible to any provider accessing the chart

How important is it to say medical provider vs. VHA?
- say it's a VA medical provider (internal to VA)

How is the indicator sent to VHA?
- basically, it's all invisible to veteran
- pulls from VBMS system from claims processor who requests exam (this is the driver)

Where does it show up in medical record?
- TBD
- interim version - chart note

Review of CAIA's recommendation
- "ok from my perspective"
- tweak title to be a little more precise
- see figma comments

**Comment by jinaryu** [comment]
Margret from VHA will take a look at Katherine's questions on Monday 8/5

**Comment by jinaryu** [comment]
@eileen-coforma some research insights on this page from our research (more to come)

- The content on this screen makes sense to most veterans and gets the 
point across that saying yes won't affect your claim, and will lead to an indicator 
being put on record (although there are opportunities to further simplify the 
content)
- Veterans also liked the option to revoke consent. 
- However, the revoking consent and VA healthcare radio buttons were confusing to some (they weren't sure if they were eligible)
- I wonder if we can make these radio buttons show up conditionally based on information we already have (ex. enrolled in VA 
healthcare, previously provided consent).

**Comment by eileen-coforma** [comment]
Margret has taken a look at the questions in the document and left comments. If we need a meeting to clarify some of the lingering edits, I can set that up. CC:@katherine-fung

**Comment by jinaryu** [comment]
@eileen-coforma some more research insights

- The veteran was confused about what an indicator is, and what kind of indicator it is compared to what would normally happen. 
- The veteran also expressed concern about who would see the indicator, and would opt "no" under the assumption that anyone at VA would be able to see it. The veteran also wondered if this would impact the gender of the person selected to conduct C&P exams, and expected that this would prolong their claim time. 
- They did express that they would like the opportunity to provide preference for their examiner's gender.
- The veteran also appreciated the ability to revoke selections on this page.

**Comment by jinaryu** [comment]
Updated [Figma screen](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims-(VA-0995)?node-id=1336-55593&t=U3lpSjKidnmOs7e2-1) with edits from SME

**Comment by katherine-fung** [comment]
Thanks for sharing the research insights @jinaryu! Which version of the consent screen did the Veteran who shared these insights see? 

(I know P1 tested the version of the consent screen before CAIA edits: https://dsva.slack.com/archives/C053UDWMH7U/p1721129356067759)

**Comment by jinaryu** [comment]
[This version! ](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=5461-48732&t=GfUsRcTMa8S3HcXJ-1) @katherine-fung

**Comment by jinaryu** [comment]
Closing this ticket - edits have been finalized by CAIA and SMEs
